mParticle-Apple-SDK/mParticle.m

```diff
@@ -159,6 +159,89 @@ - (NSString *)description {
 @implementation MPDataPlanOptions
 @end
 
+@implementation MPRokt
+
+- (void)selectPlacements:(NSString * _Nullable)identifier
+              attributes:(NSDictionary<NSString *, NSString *> * _Nullable)attributes {
+    NSDictionary<NSString *, NSString *> *mappedAttributes = [MPRokt mapUserAttributes:attributes mappingDictionary:@{}];
+    for (NSString *key in mappedAttributes) {
+        [[MParticle sharedInstance].identity.currentUser setUserAttribute:key value:mappedAttributes[key]];
+    }
```

Comment:
could you actually remove this from this implementation and wait for it to be its own feature?  from my understanding, we can't test it end to end yet, so we might as well try to get the PR into development first